src/WebJobs.Extensions.DurableTask.CodeGeneration/WebJobs.Extensions.DurableTask.CodeGen.SourceGenerator/Utils/TypeExtensions.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#nullable enable
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Reflection;
+
+namespace WebJobs.Extensions.DurableTask.CodeGeneration.SourceGenerator.Utils
+{
+    /// <summary>
+    /// Extension methods for <see cref="Type"/>.
+    /// </summary>
+    public static class TypeExtensions
```

Comment:
Why do we need these utilities?

Reply:
1. ```ToCSTypeName``` and ```GetPrettyName``` are used for getting a string representation of the type that can be injected into C# code.
2. The 'Nullable' ones can be removed as they are not being used.